pyprof/prof/softmax.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, d):
         self.op_ = op
         self.args = args
 
-        assert (mod == "torch.nn.functional")
+        assert (mod in ["torch", "torch.nn.functional"])
```

Comment:
Can you link the documentation for torch.log_softmax?
I am not finding it.

Reply:
It exists but seems to be undocumented, which is probably why I never encountered it before.

Reply:
verified by pytorch team